src/utils/stripLocationInfo.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Returns a copied version of the node recieved without location attributes.
+ * @param  {Object}  node
+ * @return {Object}  node
+ */
+export default function stripLocationInfo(node) {
```

Comment:
There already exists an utility that fills the same purpose: `isEqualAst()`

It will take care of comparing two AST nodes ignoring location info and also comments.